tasks/task02/triangle_area.py

```diff
@@ -0,0 +1,56 @@
+import math

```

Comment:
Please add a task description as a doc string at the top.
"""
Task description. 
"""

Reply:
Added